ChangeLog.md

```diff
@@ -20,6 +20,13 @@ See docs/process.md for more on how version tagging works.
 
 4.0.22 (in development)
 -----------------------
+- It is now possible to load emscripten-generated code directly into an Audio
+  Worklet without using the `-sAUDIO_WORKLET` setting (which depends on shared
+  memory and `-sWASM_WORKERS`). To do this simply build with
+  `-sENVIRONMENT=worklet`. In this environment, because audio worklets don't
+  have a fetch API, you will need to either using `-sSINGLE_FILE` to embed the
+  Wasm file, or use a custom `instantiateWasm` callback while performing
```

Comment:
This requirement seems important enough to put in settings.js, I think?

Reply:
Added to the `site/source/docs/api_reference/wasm_audio_worklets.rst` docs